webpack/assets/javascripts/react_app/components/charts/StatisticsChartsList.js

```diff
@@ -26,7 +27,7 @@ class StatisticsChartsList extends React.Component {
   render() {
     const noDataMsg = __('No data available').toString();
     const tip = __('Expand the chart').toString();
-    const charts = this.props.charts.map(chart => {
+    const charts = map(this.props.charts, chart => {
```

Comment:
In this case, I think it is better to use the `_.map(...)` syntax to make the lodash dependency more explicit.